pkg/drivers/kic/oci/oci.go

```diff
@@ -107,6 +107,24 @@ func PrepareContainerNode(p CreateParams) error {
 	return nil
 }
 
+func hasMemorySwapCgroup() bool {
+	memcgSwap := true
+	if runtime.GOOS == "linux" {
+		var memoryswap string
+		if cgroup2, err := IsCgroup2UnifiedMode(); err == nil && cgroup2 {
+			memoryswap = "/sys/fs/cgroup/memory/memory.swap.max"
+		} else {
+			memoryswap = "/sys/fs/cgroup/memory/memsw.limit_in_bytes"
+		}
+		if _, err := os.Stat(memoryswap); os.IsNotExist(err) {
+			// requires CONFIG_MEMCG_SWAP_ENABLED or cgroup_enable=memory in grub
+			klog.Warning("Your kernel does not support swap limit capabilities or the cgroup is not mounted.")
+			memcgSwap = false
+		}
```

Comment:
one internal user told me they can not apply the memory limit either (I have logs in an issue)
I think we should skip the Memory limit too if this file doesnt exists
```
cat /sys/fs/cgroup/memory/memory.limit_in_bytes
```

Reply:
Added as PR #10507

Reply:
that one only checks for 
```
if _, err := os.Stat("/sys/fs/cgroup/memory/memsw.limit_in_bytes");
```

we need to check for  `cat /sys/fs/cgroup/memory/memory.limit_in_bytes`
to not add memory limits ( I will add a PR for that)